source/includes/gcp/_target_proxies.md

```diff
@@ -9,7 +9,7 @@ Target proxies route incoming request to either a URL map (HTTP(S) load balancin
 ```shell
 curl -X GET \
    -H "MC-Api-Key: your_api_key" \
-   "https://cloudmc_endpoint/v1/services/gcp/test-area/targetproxies"
+   "https://cloudmc_endpoint/v1/services/gcp/test-area-oox/targetproxies"
```

Comment:
Sorry, the `-oox` is only needed in the project name when the URL is a googleapis.com one, when it's cloudmc, it's `test-area`

Reply:
LOL

Reply:
506af5ee288e48c8f8c6bab9e49bbb2e84808081